compiler-plugin/src/main/kotlin/arrow/meta/quotes/ClassOrObject.kt

```diff
@@ -13,12 +13,97 @@ import org.jetbrains.kotlin.psi.psiUtil.getValueParameters
 import org.jetbrains.kotlin.psi.psiUtil.modalityModifierType
 import org.jetbrains.kotlin.psi.psiUtil.visibilityModifierType
 
+/**
+ * An extension function of [Meta] and inheriting from [ExtensionPhase], [classOrObject] was designed to to feed in
+ * virtually any kind of [KtClass] predicate, followed by mapping a function that takes the desired [Scope] of our
+ * [KtClass] to change whatever PSI elements desired.
+ *
+ * @param [match] designed to to feed in any kind of [KtClass] predicate returning a [Boolean]
```

Comment:
All these block tags `@param`, `@return` etc should all be grouped at the bottom of the KDoc, that is done by convention.

Example: [Arrow Fx Concurrent](https://github.com/arrow-kt/arrow/blob/c3fa9ec8567da5bfbd9ecaac2bc055c908dfac00/modules/fx/arrow-fx/src/main/kotlin/arrow/fx/typeclasses/Concurrent.kt#L138).

I think that might also fix the newline issue

Reply:
ahh - thank you so much Simon! This is helpful.